cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -1669,7 +1669,7 @@ reportCommand =
     , commandSynopsis = "Upload build reports to a remote server."
     , commandDescription = Nothing
     , commandNotes = Just $ \_ ->
-        "You can store your Hackage login in the ~/.config/cabal/config file\n"
+        "You can store your Hackage login in the ~/.config/cabal/config file (the %APPDATA%\\cabal\\config file on Windows)\n"
```

Comment:
This line looks a bit long; can it be split in two?

Reply:
Which does it mean a length of a source code or one of printed text?

```suggestion
        "You can store your Hackage login in the ~/.config/cabal/config file\n(the %APPDATA%\\cabal\\config file on Windows)\n"
```

```suggestion
        "You can store your Hackage login in the ~/.config/cabal/config file "
          ++ "(the %APPDATA%\\cabal\\config file on Windows)\n"
```

Or both?

```suggestion
        "You can store your Hackage login in the ~/.config/cabal/config file\n"
          ++ "(the %APPDATA%\\cabal\\config file on Windows)\n"
```

Reply:
I'm thinking of display in a normal-sized terminal/console window (80 characters, in particular on Windows). A quick test here suggests the string wraps, although I use a tiling WM so I don't normally get 80 column terminals. I presume fourmolu will suggest any changes needed based on source code length.